platform/mellanox/bios.mk

```diff
@@ -0,0 +1,15 @@
+# mellanox bios firmware
```

Comment:
'Mellanox BIOS Firmware' like in other files

Reply:
@liat-grozovik fixed